README.md

```diff
@@ -9,7 +9,13 @@ yarn add glimmer-redux
 yarn add rollup-plugin-glimmer-redux
 ```
 
-Open the ember-cli-build.js file and add the rollup plugin
+Open the ember-cli-build.js file and import the rollup plugin
```

Comment:
Maybe use ` (backticks) around that file name.

Reply:
Will do!